typescript/agent/vercel-agent/src/setupOtel.ts

```diff
@@ -0,0 +1,30 @@
+import { diag, DiagConsoleLogger, DiagLogLevel } from '@opentelemetry/api';
+import { NodeSDK } from '@opentelemetry/sdk-node';
+import { AlwaysOnSampler, SimpleSpanProcessor } from '@opentelemetry/sdk-trace-node';
+import { OTLPHttpProtoTraceExporter } from '@vercel/otel';
+import { env } from 'process';
+
+diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.DEBUG); // set diaglog level to DEBUG when debugging
+
+/**
+ * Setup the OpenTelemetry SDK for the application.
+ */
+export function setupOtel() {
+  const galileoConsoleUrl = env.GALILEO_CONSOLE_URL || "https://app.galileo.ai";
```

Comment:
We should be using `http://api.galileo.ai/otel/traces` instead of `app.`. Can we swap this code to something that loads the API url from an env var? We can link to the docs on this value - https://v2docs.galileo.ai/sdk-api/third-party-integrations/opentelemetry-and-openinference#configure-the-otel-endpoint